README.md

```diff
@@ -72,12 +65,40 @@ A valid (read: that follows this interface) IPLD format implementation the follo
 
 `callback` must have the signature `function (err, dagNode)`, where `err` is an Error if the function fails and `dagNode` is the dagNode that got deserialized in the process.
 
-#### `util.cid(dagNode, callback)`
+#### `util.cid(dagNode[, options], callback)`
 
 > get the CID of the dagNode
 
+Options include:
+  - version - the CID version to be used (defaults to 1)
+  - hashAlg - the hash algorithm to be used (default to the one set by the format)
+
 `callback` must have the signature `function (err, cid)`, where `err` is an Error is the function fails and `cid` is a CID instance of the dagNode.
 
+#### `util.cidSerialized(binaryBlob[, options], callback)`
```

Comment:
not to build an anti-corrosion-bike-shelter - but the name would make me think this **produces** a cid that is serialized. Maybe cidFromBinary / cidFromBlob / addCidPrefix?

Reply:
I like the idea of adding _from_, like `cidFromSerialized`.

We use the `from` in other modules, so it will fit well here.